### DISCUSSION TITLE
Question on Notable Plugin skipped Repos

### DISCUSSION THREAD

**Discussion by ROMVoid95** [lowlighter/metrics#1231] [Q&A] [answered]
does `plugin_notable_skipped` support `@org/*` to ignore all repositories of a given org?

**Comment by lowlighter** [comment, marked as answer]
Hello 👋 !

Not yet, but will eventually be supported by #1221

**Reply by ROMVoid95** [reply]
Awesome! Thanks for the reply and for this great program. Have a great day

**Reply by lowlighter** [reply]
This is currently in testing on master/main branches, you can read more about this here: [`🔕 Skipping repositories in plugins`](https://github.com/lowlighter/metrics/blob/master/source/plugins/core/README.md#-skipping-repositories-in-plugins) 👍

**Reply by ROMVoid95** [reply]
Awesome! i'll be sure to check it out and try it, thanks for all your work! it's really appreciated
